bin/zis-plugin-install.sh

```diff
@@ -164,24 +164,23 @@ handle-simple-install() {
 # Returns:
 #   0 on success, 8 on error.
 #################################################
-handle-extended-intall() {(
```

Comment:
@DivergentEuropeans I am curious why do you spawn a separate shell using `()` ?

Reply:
Not quite sure, that's how the code was before. Maybe they don't need to be ( ) which could have solved this too